code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -501,6 +501,24 @@
 		"/obj/item/weapon/mop",
 		"/obj/item/weapon/storage/bag/trash")
 
+/obj/item/weapon/storage/belt/botanist
+	name = "botany gear belt"
+	desc = "A belt used to hold most gardening supplies such as min-hoes and spades."
+	icon_state = "botanybelt"
+	item_state = "botanybelt"
+	storage_slots = 8
+	fits_max_w_class = W_CLASS_MEDIUM
+	can_only_hold = list(
+		"/obj/item/weapon/storage/bag/plants",
+		"/obj/item/weapon/reagent_containers/glass/bottle",
+		"/obj/item/weapon/reagent_containers/syringe",
+		"/obj/item/weapon/reagent_containers/dropper",
+		"/obj/item/device/analyzer/plant_analyzer",
+		"/obj/item/weapon/minihoe",
+		"/obj/item/weapon/hatchet",
+		"/obj/item/weapon/bee_net",
```

Comment:
please add ALL aquaculture-related items to the list

Reply:
maybe bucket too, it's borderline because it is a big object but it's so heavily used
wrench? for moving trays

Reply:
Also forks and wirecutters since they work as minihoes and plant clippers.

Reply:
Added the fish stuff.